src/ralph/ssl_certificates/management/commands/importssl_certificates.py

```diff
@@ -0,0 +1,68 @@
+import datetime
+import fnmatch
+import os
+from pathlib import Path
+
+from cryptography import x509
+from cryptography.hazmat.backends import default_backend
+from cryptography.x509.extensions import ExtensionNotFound
+from cryptography.x509.oid import ExtensionOID, NameOID
+from django.core.management.base import BaseCommand
+
+from ralph.assets.models.assets import AssetHolder
+from ralph.ssl_certificates.models import SSLCertificate
+
+
+class Command(BaseCommand):
+
+    help = 'Import data to application from dir'
+
+    def add_arguments(self, parser):
+        parser.add_argument('certs_dir', type=str)
+
+    def handle(self, *args, **options):
+        certs_dir = options['certs_dir']
+        for root, dirs, files in os.walk(certs_dir):
+            for filename in fnmatch.filter(files, '*.crt'):
+                san = ''
+                cert = None
+                extension = None
+                try:
+                    with open(os.path.join(root, filename)) as f:
+                        pem_data = f.read()
+                    cert = x509.load_pem_x509_certificate(
+                        pem_data.encode(), default_backend()
+                    )
+                except ValueError:
+                    print('{}/{} is not valid'.format(root, filename))
```

Comment:
use ``self.stderr.write`` instead of ``print``

Reply:
I've add it